packages/docs/src/tags/heading.tsx

```diff
@@ -14,7 +14,7 @@ const Heading: React.SFC<IHeadingTag> = ({ level, route, value }) =>
     React.createElement(
         `h${level}`,
         { className: "docs-title" },
-        <a className="docs-anchor" key="anchor" name={route} />,
```

Comment:
wait I think this attribute is critical to functionality! there's a selector that checks the `name` attr (although something is currently broken #1682)

Reply:
`name` is not a valid HTML anchor attribute according to the latest `lib.d.ts`

Reply:
`data-name` then please